### PR TITLE
Add EUS upgrade on stable 4.10 to stable 4.12

### PR DESCRIPTION
**Pull request by shellyyang1989**

### CODE REVIEW

**Comment by shellyyang1989** [comment]
cc @liangxia @wzheng1 for review. Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-eus-4.10.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_INTERMEDIATE_TAGS: '@4.11 and @upgrade-sanity and @aws-ipi and @network-openshiftsdn
+        and not @fips'
+      E2E_RUN_LATEST_TAGS: '@4.12 and @upgrade-sanity and @aws-ipi and @network-openshiftsdn
+        and not @fips'
+      UPGRADE_CHECK_RUN_TAGS: '@4.12 and @upgrade-check and @aws-ipi and @network-openshiftsdn
+        and not @fips'
+      UPGRADE_PRE_RUN_TAGS: '@4.10 and @upgrade-prepare and @aws-ipi and @network-openshiftsdn
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-eus-4.10.yaml:73]: network-openshiftsdn -> network-ovnkubernetes

Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-eus-4.10.yaml]: Good catch! Updated. Thank you.

**Comment by wzheng1** [comment]
Thanks for implementing this upgrade path, Shelly!

**Pull request merged**